### PR TITLE
Rename security.knative.dev -> policy.run.google.cloud.com

### PR DESCRIPTION
**Pull request by yolocs**
Fixes https://github.com/google/knative-gcp/issues/726

## Proposed Changes

- API group name and where it's used
- Go package name
- Installation namespace and labels
- (Piggybacking) Allow controller service account to update k8s Event

**Release Note**

```release-note
Rename security.knative.dev -> policy.run.google.cloud.com
```

### CODE REVIEW

**Comment by yolocs** [comment]
FYI @Harwayne

File: config/policy/201-clusterrole.yaml [hunk 1]
```diff
@@ -61,6 +61,13 @@ rules:
   verbs:
     - get
 
+- apiGroups: [“”]
```
Comment (Harwayne) [inline config/policy/201-clusterrole.yaml]: This looks unrelated to the renaming. Is it here intentionally?

Comment (yolocs) [reply config/policy/201-clusterrole.yaml]: It is intentionally. During debugging I found the controller needs to have Event permission. So piggybacking here :D

Comment (Harwayne) [reply config/policy/201-clusterrole.yaml]: For changes of this size (almost three hundred files...), I prefer to not piggy back things like this. 

If you still want to, make sure to mention it in the PR's description. My reasoning is that if we need to roll this back (probably not for this particular PR, but somewhat common for large renaming changes), we don't want a functionality regression at the same time as a large name change. And based on the current description, I wouldn't expect any functionality changes, just name changes. So someone might do a straight rollback and not realize the problem.

Comment (yolocs) [reply config/policy/201-clusterrole.yaml]: Ack. Description updated.
